data-processing/pdf/process_pdf.py

```diff
@@ -1,243 +0,0 @@
-import logging
```

Comment:
I do get the reasoning behind deleting this... I would suggest though maybe not including it in this PR to keep this PR smaller and to make it easier to revert the deletion if it turns out we want this file for whatever reason. Especially because we're so rarely in this code, it feels a little scarier to me to pull in unrelated maintenance stuff. But I leave it up to you!